pkg/config/config.go

```diff
@@ -130,21 +132,48 @@ func (c *ConfigInfo) writeToFile() error {
 }
 
 // SetConfiguration modifies Odo configurations in the config file
-// as of now only being used for updatenotification
-func (c *ConfigInfo) SetConfiguration(parameter string, value bool) error {
+// as of now being used for timeout, updatenotification
+func (c *ConfigInfo) SetConfiguration(parameter string, value interface{}) error {
 	switch parameter {
+
+	case "timeout":
+		typedval, ok := value.(int)
+		if typedval == 0 {
+			fmt.Println("Cannot set timeout to 0 setting it to the minimum value(1sec)")
```

Comment:
I am of the opinion that we should rather error out here instead of taking decision on behalf of the user..

Also, this would be nice to have in the cmd help